csrc/apis/csharp/src/MMDeploySharp/APIs/Classifier.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace MMDeploySharp
```

Comment:
I think `namespace mmdeploy {namespace csharp {}}` is better

Reply:
Found It is common that package name is same with root namespace.